src/Commands.pm

```diff
@@ -1431,6 +1437,255 @@ sub cmdConf {
 	}
 }
 
+sub cmdxConf {
+	my ($cmd, $args) = @_;
+	my ($file,$file2,$found,$key,$oldval,$shopname,$type,$value, $name, %inf_hash, %ctrl_hash);
+	if ($cmd eq 'sconf') {
+		($key, $value) = $args =~ /(name)(?:\s)(.*)/;
+		$shopname = 1 if $key eq 'name';
+	}
+	($key, $value) = $args =~ /([\s\S]+?)(?:\s)([\-\d\.]+[\s\S]*)/ if !$shopname;
+	$key = $args if !$key;
+	$key =~ s/^\s+|\s+$//g;
+	debug "extracted from args: KEY: $key, VALUE: $value\n";
+	if (!$key) {
+		error "Syntax Error in function '$cmd'. Not found <key>\n".
+				"Usage: $cmd <key> [<value>]\n";
+		return;
+	}
+	if ($cmd eq 'iconf') {
+		%inf_hash  = %items_lut;
+		%ctrl_hash = %items_control;
+		$file = 'items_control.txt';
```

Comment:
You're not respecting command line arguments, if they're passed. This is very bad.